src/plugins/mf_priority.cpp

```diff
@@ -61,19 +69,12 @@ int64_t priority_calculation (flux_plugin_t *p,
     if (urgency == FLUX_JOB_URGENCY_EXPEDITE)
         return FLUX_JOB_PRIORITY_MAX;
 
-    // search element in map of maps by key
-    it = users.find (userid);
+    bank = static_cast<char *> (flux_jobtap_job_aux_get (
+                                                    p,
+                                                    FLUX_JOBTAP_CURRENT_JOB,
+                                                    "mf_priority:bank"));
```

Comment:
Be sure to check for failure here. It is unlikely, since all jobs have `mf_priority:bank` set before they pass validation. However, if `bank == NULL`, my guess that will lead to a later crash, which will take out the flux broker, so better to catch "unlikely" errors.

My guess is if `mf_priority:bank` isn't found, then the job should have an exception raised.

Reply:
Sounds good. I just pushed up a `fixup` commit that checks if `bank == NULL`, and if it is, it raises an exception on the job saying that the bank could not be found, and sets the priority to 0. The same exception is raised in `inactive_cb ()`.